lfs.c

```diff
@@ -8,6 +8,23 @@
 #include "lfs.h"
 #include "lfs_util.h"
 
+// Configuration Sanity Check
+#if (LFS_NAME_MAX <= 0) || (LFS_NAME_MAX > 1022)
+#error "LFS_NAME_MAX must be in the range (0, 1022]"
+#endif
+
+#if (LFS_FILE_MAX <= 0) || (LFS_FILE_MAX > 4294967296)
+#error "LFS_FILE_MAX must be in the range (0, 4294967296]"
+#endif
```

Comment:
Are you sure this is `4294967296` (AKA `2^32`) and not 4294967295 (`UINT32_MAX`)?

Reply:
I imagine your proposed `4294967295` is correct, but I was just copying from [the documentation](https://github.com/littlefs-project/littlefs/blob/f77214d1f0a8ccd7ddc7e1204fedd25ee5a41534/lfs.h#L55). Once we confirm thats the correct value, I can update both locations.

Reply:
Ah woops, yes this is a typo. 4294967295 (UINT32_MAX) should be the correct value.

Reply:
addressed in c531a5e88f09e71f9487